install.zsh

```diff
@@ -8,16 +11,16 @@ NVIM_HOME="$(realpath ~)/.config/nvim"
 ROOT=$(realpath ../)
 CONFIG=$(realpath ./config)
 
-# clone NvChad along side this folder
+# Clone NvChad along side this folder
 if [ ! -d "../NvChad" ]
 then
   echo "🚗 cloning NvChad"
-  git clone https://github.com/NvChad/NvChad $ROOT --depth 1
+  git clone https://github.com/NvChad/NvChad $ROOT/NvChad --depth 1
```

Comment:
I had to add a folder name here otherwise it was trying to unstall the repo files into that folder, not create a new folder. Was getting "dotfiles" folder already exists.

Reply:
derp...thanks for fixing